examples/job/expansions/README.md

```diff
@@ -0,0 +1,237 @@
+job "jobexample-banana" created
+job "jobexample-cherry" created
+```
+
+Now, check on the jobs:
+
+```console
+$ kubectl get jobs -l app=jobexample
+JOB                CONTAINER(S)   IMAGE(S)   SELECTOR                               SUCCESSFUL
+jobexample-apple   c              busybox    app in (jobexample),item in (apple)    1
+jobexample-banana  c              busybox    app in (jobexample),item in (banana)   0
+jobexample-cherry  c              busybox    app in (jobexample),item in (cherry)   0
+```
+
+Here we use the `-l` option to select all jobs that are part of this
+group of jobs.  (There might be other unrelated jobs in the system that we
+do not care to see.)
+
+We can check on the pods as well using the same label selector:
+
+```console
+# Now check on their pods
+$ kubectl get pods -l app=jobexample
+NAME                      READY     STATUS      RESTARTS   AGE
+jobexample-apple-43cz3    0/1       Completed   0          1m
+jobexample-banana-bv4s8   0/1       Completed   0          1m
+jobexample-cherry-ml2rv   0/1       Completed   0          1m
+```
+
+There is not a single command to check on the output of all jobs at once,
+but looping over all the pods is pretty easy:
+
+```console
+# Get logs of all the pods.
+$ for p in $(kubectl get pods -l app=jobexample -o name | cut -f 2 -d "/")
+do
+  kubectl logs $p
+done
+Processing item apple
+Processing item banana
+Processing item cherry
+```
+
+## Multiple Template Parameters
+
+In the first example, each instance of the template had one parameter, and that parameter was also
+used as a label.  However label keys are limited in [what characters they can
+contain](labels.md#syntax-and-character-set).  If your parameters cannot be limited to
+this character set or if you have multiple varying parameters, then you can use a pattern
+like this:
+
+```python
+#! /usr/bin/python
+# -*- coding: utf-8 -*-
+
+params = [
+                {
+                        "name": "apple",
+                        "filename": r"./apple.txt",
+                        "language": "en",
+                },
+                {
+                        "name": "banana",
+                        "filename": r"./香蕉.txt",
+                        "language": "zh",
+                },
+                {
+                        "name": "cherry",
+                        "filename": r"../вишня.doc",
+                        "language": "ru",
+                }
+        ]
+
+for p in params:
+  print (
+"""
+apiVersion: extensions/v1beta1
+kind: Job
+metadata:
+  name: jobexample-%(name)s
+spec:
+  selector:
+    matchLabels:
+      app: jobexample
+      item: %(name)s
+  template:
+    metadata:
+      name: jobexample
+      labels:
+        app: jobexample
+        item: %(name)s
+    spec:
+      containers:
+      - name: c
+        image: busybox
+        command: ["sh", "-c", "echo Processing item %(filename)s in language %(language)s && sleep 5"]
+      restartPolicy: Never
+---
+""" % p)
```

Comment:
I think we should provide at least the few necessary steps to actually run this example. So that a user can just hit download and copy&paste the command and voila have the job running in k8s. Currently this sample is ok, but it's quite raw. So either not include it or improve it, imho. I'd personally go the latter way.